src/bridge/layouts/mod.rs

```diff
@@ -64,20 +64,20 @@ fn append_modifiers(
     if result == "<" {
         result = "lt".to_string();
         special = true;
-    }
+    }    
 
     if shift {
         special = true;
         result = format!("S-{}", result);
     }
-    if ctrl {
+	if ctrl && !alt {
```

Comment:
As much as it is a pain, I don't think its a good idea to have both sets of changes in this PR. I added a comment on the other PR about this code and we should iterate on them separately.

Reply:
I'll update the branch later to remove this change from the PR.